packages/bench/index.js

```diff
@@ -105,6 +106,15 @@ async function runRollup(item) {
 for (const suite of suites) {
   const bench = new Bench({ time: 100, iterations: suite.benchIteration ?? 10 })
 
+  // Check if inputs have been initialized
+  for (const input of suite.inputs) {
```

Comment:
When testing, this would crash hard if `just setup-bench` wasn't ran, so added this little check.